apps/meteor/ee/client/omnichannel/slaPolicies/RemoveSlaButton.tsx

```diff
@@ -11,18 +11,14 @@ type Props = {
 };
 
 function RemoveSlaButton({ _id, reload }: Props): ReactElement {
-	const removeSLA = useMethod('livechat:removeSLA');
+	const removeSLA = useEndpoint('DELETE', `/v1/livechat/sla/${_id}`);
 	const setModal = useSetModal();
 	const dispatchToastMessage = useToastMessageDispatch();
 	const t = useTranslation();
 	const slaRoute = useRoute('omnichannel-sla-policies');
 
 	const handleRemoveClick = useMutableCallback(async () => {
-		try {
-			await removeSLA(_id);
-		} catch (error) {
-			console.log(error);
-		}
+		await removeSLA();
```

Comment:
Wasn't the try/catch required here? Does useEndpont and useMethod handle errors differently?

Reply:
Since we're now using the `useEndpoint()` here, we don't have to manually handle the API errors, because those hooks have their own mechanisms for error handling.